operations/sat/sat_in_csm.md

```diff
@@ -45,6 +45,8 @@ SAT, all files and directories are created with the appropriate permissions.
 
 For full SAT documentation, refer to the [*HPE Cray EX System Admin Toolkit (SAT) Guide*](https://cray-hpe.github.io/docs-sat/).
 
+Most `sat` subcommands depend on services or components from other products in the HPE Cray EX software stack. For more details, refer to the [*SAT Dependencies*](https://github.com/Cray-HPE/docs-sat/blob/main/docs/about_sat/dependencies.md).
```

Comment:
I just noticed this points at the main branch. It would be best to point at the branch of docs-sat which corresponds to the version of SAT in CSM 1.5, which is SAT 2.6.

```suggestion
Most `sat` subcommands depend on services or components from other products in the HPE Cray EX software stack. For more details, refer to the [*SAT Dependencies*](https://github.com/Cray-HPE/docs-sat/blob/release/2.6/docs/about_sat/dependencies.md).
```